viewlib.php

```diff
@@ -261,8 +261,15 @@ function bigbluebuttonbn_view_render_recordings(&$bbbsession, $enabledfeatures,
     );
     // If there are meetings with recordings load the data to the table.
     if ($bbbsession['bigbluebuttonbn']->recordings_html) {
+        $recordingshtml = '';
+        // If opencast recoding config is set and checks if opencast is available for the current course.
```

Comment:
I would propose to add the constraint that the OC plugin has to be installed and the option has to be enabled here once more:
```suggestion
        // If block_opencast is installed and the option to show the recordings from Opencast is enabled
        // and there is an Opencast series in this course.
```